jestSupport/preprocessor.js

```diff
@@ -26,6 +26,6 @@ module.exports = {
   getCacheKey: createCacheKeyFunction([
     __filename,
     path.join(__dirname, '../packager/transformer.js'),
-    path.join(__dirname, '../node_modules/babel-core/package.json'),
+    path.join(require.resolve('babel-core')),
```

Comment:
Sure I will :) 
I didn't manage to resolve to package.json using require.resolve options, so I have to propositions:
- not very pretty..., replace the `index.js` in the string returned by the require.resolve with `package.json`
- try both paths `'../node_modules/babel-core/package.json'` and `'../../node_modules/babel-core/package.json'` and select the existing one